misc/python/materialize/buildkite_insights/segfaults/find.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import time
+
+from materialize.buildkite_insights.util import buildkite_api
+
+
+def main() -> None:
```

Comment:
Optional: Consider using `fetch_builds` and iterating over the possibly three relevant pipelines.